fabfile.py

```diff
@@ -115,3 +115,58 @@ def setup(*args, **kwargs):
 
     # Needs to occur after base setup data has been loaded
     load_gis_data()
+
+
+def clear_cache():
+    local('redis-cli flushall')
+
+
+def warmup_cache(
+        server_location, username, password, client_id, client_secret):
+    """Warm up the cache"""
+    import requests
+    import logging
+    logging.basicConfig(level=logging.DEBUG)
+
+    def _get_url(stub):
+        return "{}{}".format(server_location, stub)
+
+    def login():
+        data = {
+            'username': username,
+            'password': password,
+            'grant_type': 'password',
+            'client_id': client_id,
+            'client_secret': client_secret
+        }
+        headers = {
+            'Accept': 'application/json'
+        }
+        resp = requests.request(
+            "POST", url=_get_url("/o/token/"), data=data,
+            headers=headers
+        )
+        if resp.status_code == 200:
+            data = resp.json()
+            return "{} {}".format(data['token_type'], data['access_token'])
+        else:
+            raise ValueError(resp.content)
+
+    def prod_api(token):
+        headers = {
+            'Authorization': token,
+            'Accept': 'application/json, */*',
+            'Accept-Encoding': 'gzip'
+        }
+        urls = [
+            "/api/gis/coordinates/?page_size=9000",
+            "/api/gis/county_boundaries/?page_size=47",
```

Comment:
Remove this page size too; it has no effect ( overridden by a custom pagination class in the view ).